src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -12,6 +14,49 @@ class AuthApplicationSection extends React.Component {
     let content;
     const verifiedAccountType = 3;// verified ID.me accounts are type 3
     const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
+    const savedForms = this.props.userProfile.savedForms;
+    const hasSavedForms = !!savedForms;
+    
+    //TODO:  (ceh) verify edu and burial form codes
+    const formTitles = {
```

Comment:
Yeah, I think we need to make this separate. I think we also want to figure out what the best way to tell people to keep this list up to date.

Off the top of my head: throw a console error when you load a form with a `formId` that isn't in this list and throw a Sentry error if you're on the profile page and there's a form in the list that doesn't have an entry.

@cvalarida Other ideas?